src/batch-client/index.ts

```diff
@@ -635,6 +644,15 @@ export class ZimbraBatchClient {
 			return normalized;
 		});
 
+	public searchAppointments = (options: SearchOptions) =>
```

Comment:
Does the other `search` function not work for appointments? This one looks very similar.